### PR TITLE
Show API usage by Key

### PR DESCRIPTION
**Pull request by forevermatt**
### Added
- Add a way for API Owners to see their API's usage on a per-key basis

### CODE REVIEW

**Comment by forevermatt** [comment]
To see which direct dependencies were updated this time, see [this commit](https://github.com/silinternational/developer-portal/pull/178/commits/85cf268734623774965eebadbbefe046784372a8#diff-c5b2f4dd53d06148672c7d8570f36042923d13b259f6ef4d9fbaf497209b7523), since we weren't previously tracking those in a way that was easy to see diffs of.

File: application/protected/models/Api.php [hunk 1]
```diff
@@ -2,11 +2,6 @@
 namespace Sil\DevPortal\models;
 
 use Sil\DevPortal\components\ApiAxle\Client as ApiAxleClient;
-use Sil\DevPortal\models\ApiVisibilityDomain;
-use Sil\DevPortal\models\ApiVisibilityUser;
-use Sil\DevPortal\models\Event;
-use Sil\DevPortal\models\Key;
-use Sil\DevPortal\models\User;
```
Comment (forevermatt) [inline application/protected/models/Api.php:9]: These were unneeded.

File: application/protected/components/LinksManager.php [hunk 2]
```diff
@@ -155,6 +155,15 @@ public static function getApiDetailsActionLinksForUser($api, $user)
                 'signal'
             );
             
+            $actionLinks[] = new ActionLink(
+                array(
```
Comment (Schparky) [inline application/protected/components/LinksManager.php:159]: Still using `array` in this project?

**Review by Schparky — APPROVED** [review]

File: application/protected/components/LinksManager.php (hunk 2 above)
Comment (forevermatt) [reply application/protected/components/LinksManager.php:159]: Copy-paste... _sigh_

**Pull request merged**